hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -546,58 +543,49 @@ def check_ssh_timeout_config(reason=''):
+                    else:
+                        checks[1] = True
+                except ValueError:
+                    raise ValueError('ClientAliveCountMax should be an integer')
+    return all(checks)
 
 
 def check_unowned_files(reason=''):
     """
     Ensure no unowned files or directories exist
     """
     raise CommandExecutionError('Module disabled due to performance concerns')
-    unowned_files = _execute_shell_command("df --local -P | awk 'NR!=1 {print $6}' | xargs -I '{}' find '{}' -xdev -nouser 2>/dev/null", python_shell=True).strip()
-    unowned_files = unowned_files.split('\n') if unowned_files != "" else []
-    # The command above only searches local filesystems, there may still be compromised items on network
-    # mounted partitions.
-    # Following command will check each partition for unowned files
-    unowned_partition_files = _execute_shell_command("mount | awk '{print $3}' | xargs -I '{}' find '{}' -xdev -nouser 2>/dev/null", python_shell=True).strip()
-    unowned_partition_files = unowned_partition_files.split('\n') if unowned_partition_files != "" else []
-    unowned_files = unowned_files + unowned_partition_files
-    return True if unowned_files == [] else str(list(set(unowned_files)))
 
 
 def check_ungrouped_files(reason=''):
     """
     Ensure no ungrouped files or directories exist
     """
     raise CommandExecutionError('Module disabled due to performance concerns')
-    ungrouped_files = _execute_shell_command("df --local -P | awk 'NR!=1 {print $6}' | xargs -I '{}' find '{}' -xdev -nogroup 2>/dev/null", python_shell=True).strip()
-    ungrouped_files = ungrouped_files.split('\n') if ungrouped_files != "" else []
-    # The command above only searches local filesystems, there may still be compromised items on network
-    # mounted partitions.
-    # Following command will check each partition for unowned files
-    ungrouped_partition_files = _execute_shell_command("mount | awk '{print $3}' | xargs -I '{}' find '{}' -xdev -nogroup 2>/dev/null", python_shell=True).strip()
-    ungrouped_partition_files = ungrouped_partition_files.split('\n') if ungrouped_partition_files != "" else []
-    ungrouped_files = ungrouped_files + ungrouped_partition_files
-    return True if ungrouped_files == [] else str(list(set(ungrouped_files)))
 
 
 def check_all_users_home_directory(max_system_uid):
     """
     Ensure all users' home directories exist
     """
-
-    max_system_uid = int(max_system_uid)
-    users_uids_dirs = _execute_shell_command("cat /etc/passwd | awk -F: '{ print $1 \" \" $3 \" \" $6 \" \" $7}'", python_shell=True).strip()
-    users_uids_dirs = users_uids_dirs.split('\n') if users_uids_dirs else []
+    with open('/etc/passwd', 'r') as passwd:
+        lines = passwd.readlines()
+        users_uids_dirs = [' '.join([item.split(':')[0], item.split(':')[2], item.split(':')[5], item.split(':')[6].strip()]) for item in lines]
```

Comment:
Maybe, for clarity:
```suggestion
        items = item.split(':')[0], item.split(':')[2], item.split(':')[5], item.split(':')[6].strip()
        users_uids_dirs = [' '.join(items) for item in lines]
```
You decide, this is just a suggestion.